src/main/java/nl/ordina/jobcrawler/model/Skill.java

```diff
@@ -10,7 +10,6 @@
 import java.util.UUID;
 
 @Entity
-@ToString
```

Comment:
Reason for removing @ToString?

Reply:
ToString was already provided in skill.java so using the annotation would mean a duplicate for the tostring method